waltz-data/src/main/java/com/khartec/waltz/data/logical_flow/LogicalFlowDao.java

```diff
@@ -140,6 +138,30 @@ public LogicalFlow findBySourceAndTarget(EntityReference source, EntityReference
     }
 
 
+    public List<LogicalFlow> findBySourcesAndTargets(List<Tuple2<EntityReference, EntityReference>> sourceAndTargets) {
+        if(sourceAndTargets.size() == 0) {
```

Comment:
v. minor - but `.isEmpty()` should be used instead of `size() == 0` (internal impl if often much faster)

Reply:
agreed